src/views/StandaloneDirectorsFiling.vue

```diff
@@ -625,9 +637,19 @@ export default {
           }
           filing = res.data.filing
           this.haveChanges = false
-        }).catch(error => {
+        }).catch(async error => {
           if (error && error.response && error.response.status === PAYMENT_REQUIRED) {
-            this.paymentErrorDialog = true
+            if (error.response.data && error.response.data.errors) {
+              const msgCode = error.response.data.errors.find(x => x.payment_error_type.startsWith('BCOL'))
+              if (msgCode) {
+                const errObj = await this.getErrorObj(msgCode.payment_error_type)
```

Comment:
Looks like might be some potential for mixin logic as it is duplicated across components